pkg/descriptions.json

```diff
@@ -1042,6 +1042,15 @@
         "System.Buffers.Text.Utf8Formatter"
     ]
   },
+  {
+    "Name": "Microsoft.Compatibility.AsyncInterfaces",
+    "Description": "Provides interfaces and helper types for consuming and producing async enumerables on .NET Standard 2.0. This package is not required when targeting .NET Standard 2.1 or .NET CoreApp 3.0 as they already have support for these types inbox.",
```

Comment:
Description says 2.0, but above you changed it to 2.1.

Reply:
2.0 is the right one, the above change was to be able to cross compile for 2.1 which we also need in order to produce a facade for that TFM.